src/java/org/jivesoftware/openfire/plugin/rest/entity/MUCRoomEntity.java

```diff
@@ -17,6 +17,7 @@
 package org.jivesoftware.openfire.plugin.rest.entity;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import org.glassfish.jersey.internal.guava.MoreObjects;
```

Comment:
This only does string concatenation? Intellij will auto-generate a toString() method that generates about as much code, but does not pull in a third party dependency. If this really is only string concat, I'd prefer that.

Reply:
It's not a new external dep, since it's Jersey, but it would make it harder to switch away in the future. I'll concatenate manually. :+1: